pages/common/gnomon.md

```diff
@@ -4,8 +4,24 @@
 
 - Use UNIX (or DOS) pipes to pipe the stdout of any command through gnomon:
 
-`npm test | gnomon`
+`{{npm test}} | gnomon`
 
-- Use command-line options to adjust gnomon's behavior for any command:
+- Show number of seconds since the start of the process:
 
-`{{any command}} | gnomon --type=elapsed-total --high=8.0`
+`{{npm test}} | gnomon --type=elapsed-total`
+
+- Show an absolute timestamp in UTS:
```

Comment:
What's `UTS`?

Reply:
It should be UTC. I need to stop making typos.